classes/monograph/ChapterDAO.inc.php

```diff
@@ -217,6 +246,80 @@ function resequenceChapters($monographId = null) {
 	function getInsertId() {
 		return $this->_getInsertId('submission_chapters', 'chapter_id');
 	}
+
+	/**
+	 * @copydoc PKPPubIdPluginDAO::pubIdExists()
```

Comment:
Not sure a `@copydoc` will work here -- the parameter names have changed. Perhaps we need to make the parameter names more general in `pkp-lib` to avoid confusion here?